### ISSUE TITLE
Hang/Crash on Linux when doing most things

### ISSUE THREAD

**Issue by Khhs167** [BleuBleu/FamiStudio#169]
When does it crash/hang:

- Opening files
- Switching from NTSC to PAL
- Exporting audio
- Most likely even more

Other issues:

- Playback seems to stop after one frame(166 ms)
- No audio in playback

More information:
```
OS: Arch Linux x86_64 
Device: ThinkPad T440s
Kernel: 6.0.2-arch1-1
Sound: PipeWire 0.3.59
Mono: 6.12.0
Dotnet runtime: 6.0.10, 3.1.28
Install Source: Binaries from GitHub, AUR via yay
```

Hope this is sufficient information. I can give more.

Oh and no significant logs.
```
Default OpenAL audio device is 'OpenAL Soft'
Default OpenAL audio device is 'OpenAL Soft'
```

**Comment by BleuBleu** [comment]
Hi.

This seem to be a rare audio issue with pipewire. I would recommend one of two things:

1. Delete the `libopenal32.so` that comes with FamiStudio and create a symlink to the one on your machine. Obviously give the symlink the name `libopenal32.so` :)
2. Try the FlatPak version, it already does that under the hood (i think!).

-Mat

**Comment by Khhs167** [comment]
I managed to resolve the issue, although it took some tinkering.
What i did:

1. Opened `FamiStudio.exe.config`
2. Removed the target for libopenal

Making symlinks didn't work sadly, but this did.

Surprisingly easy

**Comment by BleuBleu** [comment]
Thanks for being pro active on this. Will merge the PR when I'm back home (2nd week Nov) and it will be included in the hot fix that will follow shortly after. My big list is quite long.

-Mat

**Comment by axredneck** [comment]
I had similar crash, uninstalling `pulseaudio-alsa` package solved it (see issue #79).
